formulas/github-get-archive/functions/github-get-archive

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
No documentation about what this does.

You should move the guts of `run` to here.